swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -29,9 +29,7 @@ public final class Store: ObservableObject {
   @Published private(set) var systemExtensionStatus: SystemExtensionStatus?
 #endif
 
-  // This is not currently updated after it is initialized, but
```

Comment:
Not sure what this comment was referring to, but it's not applicable any longer.